src/main/cms/cms.h

```diff
@@ -24,6 +24,14 @@
 
 #include "common/time.h"
 
+#define CMS_KEY_NONE    0
```

Comment:
`enum` will be cleaner

Reply:
Yes, i think so too. ;)

Reply:
Same here, perhaps we should refactor the CMS.

Reply:
The constants are public now ...

Reply:
OK, convinced.